pygmt/base_plotting.py

```diff
@@ -1226,7 +1226,7 @@ def contour(self, x=None, y=None, z=None, data=None, **kwargs):
         p="perspective",
         t="transparency",
     )
-    @kwargs_to_strings(R="sequence", p="sequence")
+    @kwargs_to_strings(R="sequence", c="sequence", p="sequence")
```

Comment:
It should be `sequence_comma`.
```suggestion
    @kwargs_to_strings(R="sequence", c="sequence_comma", p="sequence")
```

Reply:
Oh right. Somehow my local tests passed still...